tests/network-ovn

```diff
@@ -643,9 +656,18 @@ ovn_forward_tests() {
     echo "==> Check basic forward default target address can be set (to non-existent target address)"
     lxc network forward set ovn-virtual-network 192.0.2.1 target_address="${ip4AddressPrefix}.2"
     lxc network forward set ovn-virtual-network 2001:db8:1:2::1  target_address="${ip6AddressPrefix}::2"
-    ovn-nbctl list load_balancer | grep -cF name | grep -xF 2
+    if hasNeededAPIExtension network_allocate_external_ips; then
+      echo "==> Check basic forward default target address can be set on allocated forward (to non-existent target address)"
+      lxc network forward set ovn-virtual-network "${allocated_ipv4_address}" target_address="${ip4AddressPrefix}.3"
+      lxc network forward set ovn-virtual-network "${allocated_ipv6_address}"  target_address="${ip6AddressPrefix}::3"
+    fi
+    ovn-nbctl list load_balancer | grep -cF name | grep -xF 4
```

Comment:
Should this `ovn-nbctl list load_balancer` be in the `if`? It seems to explain why `5.0/edge` failed: https://github.com/canonical/lxd-ci/actions/runs/9447367697/job/26019032733?pr=186#step:7:1658

Reply:
Ah yes there should be 4 in the if block and 2 otherwise. Will change.

Reply:
Fixed.